ui_src/src/NotesHApp.js

```diff
@@ -98,11 +98,11 @@ function NoteForm ({ note = { title: '', content: '' }, formTitle, formAction, s
     <h3>{formTitle}</h3>
     <div className='form-row'>
       <label htmlFor='title'>Title</label>
-      <input id='title' name='title' value={title} onChange={setField('title')} />
+      <input id='title' name='title' value={title} onChange={setField('title')} data-testid='title-field' />
```

Comment:
Do we need these input elements to have data-testids?  We could reference them with the the React Testing `ByLabelText` query. (https://testing-library.com/docs/dom-testing-library/api-queries#bylabeltext)

This is a bit picky, but just thinking about what standards we'd like to promote and focus on in the RAD tools.  I'm open to discuss either way.

Thoughts?

Reply:
Good catch. This was part of trying to get this test to work. I just forgot to put it back to labels. Will do now.